CHANGELOG.md

```diff
@@ -5,8 +5,18 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- scikit-learn decision tree models
+- `DecisionTreeRegressor` model
+- `Tree` Transporter
+- Decision Tree chain
+- `decision_tree_chain.py` 
+- `DecisionTreeRegressor` Test
```

Comment:
Please remove this line

Reply:
Do you mean that I wrote the Decision tree chain twice? hmm, you're right, I did remove the second one(`decision_tree_chain.py`)